content/docs/intro/console/accounts/editions.md

```diff
@@ -1,28 +1,44 @@
 ---
-title: Plans and Editions
-
-aliases: ["/docs/console/accounts/editions/"]
+title: Pulumi Product Editions
 ---
 
-There are multiple subscription plans available to Pulumi organizations, each offering an expanding set
-of features and capabilities.
+Pulumi has multiple subscription plans available to organizations, each offering an
```

Comment:
Instead of listing it here, should we just link to https://pulumi.com/pricing? This is a subset and I worry it'll be easy for these to get out of synch.